digihel/settings.py

```diff
@@ -208,6 +208,15 @@
 WAGTAIL_SITE_NAME = "digihel"
 SITE_ID = 1
 
+WAGTAILSEARCH_BACKENDS = {
```

Comment:
What if I make this conditional on the `CI` envvar, i.e. Travis would use ES? After all it has the ES addon enabled anyway.